src/data/bedrock/EffectIdMap.php

```diff
@@ -74,6 +74,7 @@ private function __construct(){
 		//TODO: SLOW_FALLING
 		//TODO: BAD_OMEN
 		//TODO: VILLAGE_HERO
+		$this->register(EffectIds::DARKNESS, VanillaEffects::DARKNESS());
```

Comment:
Please put this line above the comment lines.

Reply:
It doesn't matter, since they are ordered by ID value anyway (though that's a dumb idea in itself).